shared/src/packages/trucks/libs/validation-schemas/truck-id-parameter.validation-schema.ts

```diff
@@ -0,0 +1,7 @@
+import joi from 'joi';
+
+const truckIdParameterValidationSchema = joi.object({
+  id: joi.number().required(),
```

Comment:
see example in businessGetParameters validation schema

Reply:
thanks